daml-script/test/src/com/digitalasset/daml/lf/engine/script/test/DevIT.scala

```diff
@@ -18,11 +17,11 @@ import org.scalatest.wordspec.AsyncWordSpec
 
 // TODO(#17366): Once daml3-script diverges from script, V1DevIT and V2DevIT may not be able to
 //     share the same code anymore.
-class DevITV1 extends DevIT(V1)
+class DevITV1 extends DevIT(LanguageMajorVersion.V1)
 
 // TODO(https://github.com/digital-asset/daml/issues/17812): re-enable this test and control its run
 //  at the bazel target level.
-//class DevITV2 extends DevIT(V2)
+//class DevITV2 extends DevIT(LanguageMajorVersion.V2)
```

Comment:
This is kind of orthogonal from this PR but I noticed while temporarily re-enabling this test that the non-qualified name makes the commented code invalid without an additional import.